python/src/naive_api_client.py

```diff
@@ -72,3 +72,21 @@ class VerificationRequest(TypedDict):
             json=request_data,
             headers=self.API_HEADERS,
         ).json()
+
+    def get_income_info_by_token(self, access_token: str) -> Any:
+        """
+        https://docs.citadelid.com/#income-verification
+        :param access_token:
+        :return:
+        """
+
+        class VerificationRequest(TypedDict):
+            access_token: str
+
+        request_data: VerificationRequest = {'access_token': access_token}
```

Comment:
was written by me like in yours code, but i think better  to write everywhere like
```
request_data = VerificationRequest(access_token=access_token)
```
but in common it doesn't matter